docs/sources/mimir/visualize/query-metric-data.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Please put this in a top-level _Query_ section. See https://github.com/grafana/mimir/issues/4509.